src/cms/index.js

```diff
@@ -0,0 +1,138 @@
+  // eslint-disable-next-line react/prop-types
+  logo,
+  // eslint-disable-next-line react/prop-types
+  flourishURL,
+  // eslint-disable-next-line react/prop-types
+  fetchDefinition,
+  // eslint-disable-next-line react/prop-types
+  fetchDefinitionUrl
+}) {
+  return (
+    <>
+      {Array.from(
+        document.querySelectorAll(`div[id^=${TYPES.HURUMAP_CARD}]`)
+      ).map(el =>
+        ReactDOM.createPortal(
+          <Visual
+            parentEl={el}
+            logo={logo}
+            fetchDefinition={fetchDefinition}
+            fetchDefinitionUrl={fetchDefinitionUrl}
+            id={el.getAttribute('data-post-id')}
+            type={el.getAttribute('data-post-type')}
+          />,
+          el
+        )
+      )}
+      {Array.from(
+        document.querySelectorAll(`div[id^=${TYPES.FLOURISH_CHART}]`)
+      ).map(el =>
+        ReactDOM.createPortal(
+          <Visual
+            parentEl={el}
+            logo={logo}
+            flourishURL={flourishURL}
+            fetchDefinition={fetchDefinition}
+            fetchDefinitionUrl={fetchDefinitionUrl}
+            id={el.getAttribute('data-chart-id')}
+            title={el.getAttribute('data-chart-title')}
+            description={el.getAttribute('data-chart-description')}
+            showInsight={el.getAttribute('data-show-insight') === 'true'}
+            insightTitle={el.getAttribute('data-insight-title')}
+            insightSummary={el.getAttribute('data-insight-summary')}
+            analysisLinkCountrySlug={el.getAttribute('data-analysis-country')}
+            analysisLinkTitle={el.getAttribute('data-analysis-link-title')}
+            dataLinkTitle={el.getAttribute('data-data-link-title')}
+            dataLinkGeoId={el.getAttribute('data-data-geo-id')}
+          />,
+          el
+        )
+      )}
+      {Array.from(
+        document.querySelectorAll(`div[id^=${TYPES.HURUMAP_CHART}]`)
+      ).map(el =>
+        ReactDOM.createPortal(
+          <Visual
+            parentEl={el}
+            fetch={fetch}
+            logo={logo}
+            fetchDefinition={fetchDefinition}
+            fetchDefinitionUrl={fetchDefinitionUrl}
+            chartId={el.getAttribute('data-chart-id')}
+            geoId={
+              // data-geo-type to support old posts
+              el.getAttribute('data-geo-type') || el.getAttribute('data-geo-id')
+            }
+            showInsight={el.getAttribute('data-show-insight') === 'true'}
+            showStatVisual={el.getAttribute('data-show-statvisual') === 'true'}
+            insightTitle={el.getAttribute('data-insight-title')}
+            insightSummary={el.getAttribute('data-insight-summary')}
+            analysisCountry={el.getAttribute('data-analysis-country')}
+            analysisLinkTitle={el.getAttribute('data-analysis-link-title')}
+            dataLinkTitle={el.getAttribute('data-data-link-title')}
+            dataLinkGeoId={el.getAttribute('data-data-geoid')}
```

Comment:
`data-data-geo-od` ... nitpick: since there are a lot of them and it's easy to make a typo, can we declare this props in `const` somewhere